code/modules/surgery/surgery_step.dm

```diff
@@ -175,3 +175,8 @@
 	if(!target_detailed)
 		var/you_feel = pick("a brief pain", "your body tense up", "an unnerving sensation")
 		target.show_message(vague_message, MSG_VISUAL, span_notice("You feel [you_feel] as you are operated on."))
+
+/datum/surgery_step/proc/display_pain(mob/living/target, pain_message, mechanical_surgery = FALSE)
+	to_chat(target, pain_message)
+	if(prob(30) && !mechanical_surgery)
+		INVOKE_ASYNC(target, /mob.proc/emote, "scream") //i have no idea what async does but i'm copying it from dismemberment
```

Comment:
you probably dont want async

Reply:
Should I just use `target.emote("scream")` then?

Reply:
I'll do that for now but if that's wrong do yell at me